ui/components/multichain/pages/connections/connections.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export const Connections = () => {
+  return <div></div>;
```

Comment:
```suggestion
  return <div />;
```

Reply:
Thanks for pointing it out @DDDDDanica but I think we can keep it as it is since this is just a layout structure.